src/tiled/tileanimationeditor.cpp

```diff
@@ -67,17 +67,19 @@ class FrameListModel : public QAbstractListModel
 
     void setFrames(const Tileset *tileset, const QVector<Frame> &frames);
     void addTileIdAsFrame(int id);
+    void setDefaultFrameTime(int duration);
     const QVector<Frame> &frames() const;
 
 private:
-    static const int DEFAULT_DURATION = 100;
+    static int DEFAULT_DURATION;
 
     void addFrame(const Frame &frame);
 
     const Tileset *mTileset;
     QVector<Frame> mFrames;
 };
 
+int FrameListModel::DEFAULT_DURATION=100;
```

Comment:
I guess you can initialise this in the constructor.

Reply:
I dont think this can be done in a constructor as its a static variable ..